packages/core/src/internal/data-grid/data-grid-render.tsx

```diff
@@ -1828,7 +1828,7 @@ function drawColumnResizeOutline(
     ctx.lineTo(yOffset, height);
 
     ctx.lineWidth = 2;
-    ctx.strokeStyle = theme.accentColor;
+    ctx.strokeStyle = theme.accentLight;
```

Comment:
we should probably blend this onto `bgCell` so that it is a solid color

Reply:
Added a blend 👍